src/glance.gleam

```diff
@@ -1471,7 +1471,18 @@ fn list(
 
     [#(t.Comma, _), #(t.RightSquare, _), ..tokens] if acc != [] ->
       Ok(#(list.reverse(acc), None, tokens))
+    [#(t.DotDot, _), #(t.RightSquare, _) as close, ..tokens] -> {
+      case discard {
+        None -> unexpected_error([close, ..tokens])
+        Some(discard) -> Ok(#(list.reverse(acc), Some(discard), tokens))
+      }
+    }
```

Comment:
Is this used? I think the test is for the case below, I'm not sure what this one is for.

Reply:
I added a test case for it.